subprojects/tooling-api/src/main/java/org/gradle/tooling/internal/consumer/loader/DefaultToolingImplementationLoader.java

```diff
@@ -84,7 +85,7 @@ public ConsumerConnection create(Distribution distribution, ProgressLoggerFactor
 
             // Adopting the connection to a refactoring friendly type that the consumer owns
             AbstractConsumerConnection adaptedConnection;
-            if (connection instanceof InternalTestExecutionConnection){
+            if (connection instanceof InternalTestExecutionConnection) {
                 adaptedConnection = new TestExecutionConsumerConnection(connection, modelMapping, adapter);
             } else if (connection instanceof StoppableConnection) {
                 adaptedConnection = new ShutdownAwareConsumerConnection(connection, modelMapping, adapter);
```

Comment:
You would use `DeprecatedConsumerConnection` here, around the `ShutdownAwareConsumerConnection`.